tests/unit/amo/utils/test_index.js

```diff
@@ -67,7 +67,19 @@ describe(__filename, () => {
     });
 
     describe('isInternal prop', () => {
-      it('returns true for a URL containing the current host', () => {
+      it('returns true for a single slash-prefixed URL', () => {
+        expect(checkInternalURL({ urlString: pathname }).isInternal).toEqual(
```

Comment:
It's not clear to me that `pathname` is a "single slash-prefixed URL", which I guess is a relative URL. I'd rather put a fixed value here like `/relative/url`